e2e/worker/worker_test.go

```diff
@@ -160,14 +166,43 @@ publish:
 	Context("deploys the counter service", func() {
 		It("svc init should succeed", func() {
 			_, err := cli.SvcInit(&client.SvcInitRequest{
-				Name:               counterServiceName,
-				SvcType:            "Worker Service",
-				Dockerfile:         "./counter/Dockerfile",
-				TopicSubscriptions: []string{fmt.Sprintf("%s:%s", workerServiceName, "processed-msg-count")},
+				Name:            counterServiceName,
+				SvcType:         "Worker Service",
+				Dockerfile:      "./counter/Dockerfile",
+				NoSubscriptions: true,
 			})
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("adds a topic and queues to push to the service", func() {
```

Comment:
An alternative would be to write add a `manifest.yml` file under `copilot/worker/manifest.yml`, like what we do with `multi-svc-app`.